src/components/cards/serviceCard.js

```diff
@@ -0,0 +1,24 @@
+import './index.css'
+const ServiceCard=()=>{
+    return <div className="serviceCard flex-V-center-VH"><div class="sq-card">
+    <img src="https://b.zmtcdn.com/webFrontend/e5b8785c257af2a7f354f1addaf37e4e1647364814.jpeg?fit=around|402:360&crop=402:360;*,*" alt="service-card" />
```

Comment:
you should wrap this img tag with a img container div and make your image responsive by adding a img-resp class on that have the img tag be width : 100% and height : 100% and then define the height and width on the wrapper div and add those object-fit and stuff to the image